roles/keepalived/templates/keepalived.conf.j2

```diff
@@ -11,12 +11,18 @@ global_defs {
 {% if keepalived_set_script_security_flag %}
     enable_script_security
 {% endif %}
+{% if keepalived_enable_mail_notification | default(false) %}
+{% if keepalived_notification_email | default([]) | length > 0 %}
     notification_email {
-        {{ keepalived_notification_email }}
+{% for email in keepalived_notification_email %}
+        {{ email }}
+{% endfor %}
     }
+{% endif %}
     notification_email_from {{ keepalived_notification_email_from }}
     smtp_server {{ keepalived_smtp_server }}
     smtp_connect_timeout 30
+{% endif %}
```

Comment:
It could make sense to end the inner if-clause at the bottom as well since the smtp related settings don't have an effect.

In this case, we could even think about getting rid of `keepalived_enable_mail_notification` entirely and activate it only if the list contains items.

```suggestion
    notification_email_from {{ keepalived_notification_email_from }}
    smtp_server {{ keepalived_smtp_server }}
    smtp_connect_timeout 30
{% endif %}
{% endif %}
```

Reply:
Good point. Removed `keepalived_enable_mail_notification` entirely.